modules/mco-update-boot-images-disable.adoc

```diff
@@ -9,6 +9,21 @@
 
 To disable the updated boot image feature, edit the `MachineConfiguration` object so that the `machineManagers` field is an empty array.
 
+[NOTE]
+====
+If you are updating a {GCP} or {AWS} cluster from {product-title} 4.18 to 4.19, and you have not configured the `managedBootImages` parameter, the update is blocked with a _This cluster is GCP or AWS but lacks a boot image configuration._ message. The update is blocked intentionally in order to alert you that the default updated boot image behavior is changing between version 4.18 and 4.19 to enable updated boot images by default.
+
+To allow the update, perform one of the following tasks:
+
+* If you do not want the updated boot image feature enabled, explicitly disable the feature by using the following procedure.
```

Comment:
nit: can we flip the order of these two list entries?  First, I think we prefer the MCO management, because it conveniently keeps things current in the cluster, and opening with our preferred option saves the folks taking that path some reading.  Second, it makes the "using the following procedure" more clearly "the following _Procedure_ subsection about editing the MachineConfiguration", when it feels like the current ordering might have some readers confuse this to mean "the `admin-acks` patch in the following list entry".